lib/jira/client/index.js

```diff
@@ -152,28 +142,27 @@ module.exports = async (jiraHost, gitHubInstallationId, logger) => {
  * Splits commits in data payload into chunks of 400 and makes separate requests
  * to avoid Jira API limit
  */
-const batchedCommitRequest = async (data, options, instance, installationId) => {
-  const filteredCommits = dedupCommits(data)
+const batchedBulkUpdate = async (data, options, instance, installationId) => {
+  const dedupedCommits = data.commits ? dedupCommits(data.commits) : []
```

Comment:
What do you think about pushing this conditional into `dedupCommits`? It would mean `dedupCommits` accepts either an empty array or a null value. I favour that only because it pushes some complexity out of this function, which already does a fair bit. Within `dedupCommits`, this case can be handled elegantly:

```javascript
const dedupCommits = (commits) => {
  return (commits || []).filter((obj, pos, arr) => {
    // ...
  })
}
```

This is a matter of preference so I'd encourage you to pick whichever approach you find most readable and maintainable (even if it differs from my suggestion).